### PR TITLE
fix(session): add session token to account delete

### PR DESCRIPTION
**Pull request by vbudhram**
Connects to https://github.com/mozilla/fxa-auth-server/issues/2312

### CODE REVIEW

**Comment by vbudhram** [comment]
blocked on https://github.com/mozilla/fxa-js-client/pull/275

**Comment by vladikoff** [comment]
@vbudhram to do final check

**Comment by vbudhram** [comment]
@shane-tomlinson r?

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -12835,12 +12644,6 @@
       "integrity": "sha1-7GIRvtGSBEIIj+D3Cyg3Iy7SyKg=",
       "dev": true
     },
-    "sigmund": {
```
Comment (shane-tomlinson) [inline npm-shrinkwrap.json]: I did an `npm run shrink` for another PR and saw the exact same set of dependencies removed and I thought I did it wrong. Not any more.

**Pull request merged**